docker-compose.yml

```diff
@@ -20,6 +20,8 @@ services:
       - pgdata:/var/lib/postgresql/data
     networks:
       - main
+    ports:
```

Comment:
I found this helpful during development, being able to connect directly with a database tool like DBeaver.